### PR TITLE
feat(tailwind boilerplate): Implementing tailwindcss

### PR DESCRIPTION
**Pull request by phill52**
I set up some boilerplate code, file structure, and added the tailwindcss plugin
if you want to set up the file structure differently let me know

### CODE REVIEW

File: .github/CONTRIBUTORS.MD [hunk 1]
```diff
@@ -3,4 +3,5 @@
 Specials thanks to the members of Stevens Blueprint for collaborating on this project
 
 <!--Please add your name (First Name Last Name)-->
-- [Miguel Merlin](https://github.com/miguel-merlin)
\ No newline at end of file
+- [Miguel Merlin](https://github.com/miguel-merlin)
+- [Phillip Anerine](https://github.com/phill52)
```
Comment (miguel-merlin) [inline .github/CONTRIBUTORS.MD:7]: Please, for next time, try to keep the changes in pr atomic. Meaning try to avoid unrelated changes out of the pr. https://mtlynch.io/code-review-love/

**Review by miguel-merlin — APPROVED** [review]
This is great! Thanks for your work!

**Comment by ezrizhu** [comment]
Please use a branch made from this repo. (just push this to a separate branch here, and change this PR to be based from that branch)
